include/config.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "logger.h"
+
+struct config_general {
+	char *configfile;
```

Comment:
Do we really need this in the config file struct?

Reply:
Same idea, it's one option, which will only be set by commandline parameter (or maybe environment variable). This should (can) not be read from the configfile, but it also serves as a placeholder for general settings, which might/will come later. 
Also it's nice for debugging, if you can dump the state/all settings of xdpw.

Reply:
I'd rather have this as a separate variable.

Reply:
ok also regarding the loglevel: the config should only contain configs for each portal?

Reply:
What do you mean?

Reply:
i would remove the loglevel from config and the configfile setting with it's corresponding structs, so only the output_name for the screecast struct would stay

Reply:
Yeah, that sounds good to me. We'll add more config options as we go.

Reply:
(Side note, it's always easier to add new config options, than to remove existing ones. Removing config options means we break user configurations.)